src/facet.ts

```diff
@@ -17,7 +17,21 @@ export interface Header extends Guide {
    */
   labelAngle?: number;
 
-  // TODO: labelPadding
+  /**
+   * Padding spacing in pixels between header labels and axes.
```

Comment:
padding? spacing? Why both?

Reply:
See https://vega.github.io/vega-lite/docs/axis.html#title for how we name things in axes.

Reply:
I already saw that.  This is pretty consistent.